bears/hypertext/HTMLLintBear.py

```diff
@@ -28,9 +28,95 @@ class HTMLLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
+                         doctype: str='Enable',
+                         entities: str='Enable',
+                         trailing_whitespace: str='Enable',
```

Comment:
most of the settings start with either ``allow`` or ``enable`` or ``force``... you should consider using that convention, and particularly for this setting we use ``allow_trailing_whitespaces`` [use git grep to check ]